game-core/src/main/java/games/strategy/engine/framework/startup/ui/InGameLobbyWatcher.java

```diff
@@ -121,17 +110,28 @@ private InGameLobbyWatcher(
             .playerCount(playerCount)
             .status(gameStatus)
             .round(gameRound)
-            .hostName(serverMessenger.getLocalNode().getName())
-            .comment(System.getProperty(LOBBY_GAME_COMMENTS))
+            .comment(SystemPropertyReader.gameComments())
             .passworded(passworded)
             .gameVersion("0")
             .build();
-    final ILobbyGameController controller =
-        (ILobbyGameController) this.remoteMessenger.getRemote(ILobbyGameController.REMOTE_NAME);
-    synchronized (postMutex) {
-      controller.postGame(gameId, gameDescription);
-    }
-    lobbyMessenger.addErrorListener(messengerErrorListener);
+
+    gameId = gameListingClient.postGame(gameDescription.toLobbyGame());
+
+    // Period time is chosen to less than half the keep-alive cut-off time. In case a keep-alive
+    // message is lost or missed, we have time to send another one before reaching the cut-off time.
```

Comment:
TCP is a best effort and weird things do happen at network. If the message is lost and we get an exception, sending another message within the keep-alive will still have the intended effect to avoid missing the cut-off. A second case, we could also still see long delays in message delivery. Eventually we have to detect the connection as no longer viable, which is where the 10s cut-off comes in.